scripts/ci/test_extensions.sh

```diff
@@ -24,7 +24,11 @@ exit_code=0
 # Disable k8s-extension temporarily: https://github.com/Azure/azure-cli-extensions/pull/6702
 # Disable alias temporarily: https://github.com/Azure/azure-cli/pull/27717
 # hybridaks is going to be deprecated: https://github.com/Azure/azure-cli/pull/29838
-ignore_list='azure-cli-ml fzf arcappliance arcdata connectedk8s k8s-extension alias hybridaks'
+# db-up is going to be deprecated: https://github.com/Azure/azure-cli/pull/29887
+ignore_list='azure-cli-ml fzf arcappliance arcdata connectedk8s k8s-extension alias hybridaks db-up'
+
+# Does not exit if az extension add fails until all extensions have been tested
+set +e
```

Comment:
`set +e` will make the script no longer fail in case of extension installation failure.

Reply:
This is similar to https://github.com/Azure/azure-cli/pull/14295#discussion_r1758602847.

Reply:
`test_extensions.sh` calls `scripts/ci/artifacts.sh` which calls `scripts/ci/build.sh`.

`build.sh` has `set -e`.

Once `-e` is set, this script fails immediately if one extension fails. `set +e` ensures all extension have been tested. Finally, line 59 returns exit_code.